nautobot/core/management/commands/generate_test_data.py

```diff
@@ -92,9 +92,9 @@ def handle(self, *args, **options):
         self.stdout.write("Creating Sites...")
         SiteFactory.create_batch(15)
         self.stdout.write("Creating LocationTypes...")
-        LocationTypeFactory.create_batch(6)  # only 6 unique LocationTypes are hard-coded presently
+        LocationTypeFactory.create_batch(7)  # only 7 unique LocationTypes are hard-coded presently
         self.stdout.write("Creating Locations...")
-        LocationFactory.create_batch(10)
+        LocationFactory.create_batch(100)
```

Comment:
Do we need 100 locations?

Reply:
I was doing it for tests and forgot to change it back sorry